utils/iio_common.c

```diff
@@ -359,7 +359,7 @@ struct iio_context * handle_common_opts(char * name, int argc,
 	} else if (!arg && backend != IIO_LOCAL)
 		fprintf(stderr, "argument parsing error\n");
 	else if (backend == IIO_URI)
-		ctx = iio_create_context(NULL, arg);
+		ctx = iio_create_context(&params, arg);
 	else
 		ctx = iio_create_context(NULL, NULL);
```

Comment:
likely not needed if there's no real need for a re-spin but adding the params in here should not arm (I guess this is local context and timeout is unlikely - if possible at all)...

Reply:
Note that passing `NULL` is functionally the same in Libiio as passing a zero-initialized `params`.